src/scripts/background/database.ts

```
@@ -62,7 +62,7 @@
 
     await downloadDatabase(api);
 
-    if (firstStart) {
+    if (firstStart && addonInstallReason && addonInstallReason["reason"] === "update" || "install") {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)